### PR TITLE
Add Club pages models

### PR DESCRIPTION
**Pull request by comzeradd**

### CODE REVIEW

**Comment by comzeradd** [comment]
Good point, but don't we need a homepage for the overall coss project unrelated to open source clubs? I'm talking about the [top-level landing page](https://m-branson.github.io/coss-os-ui/#artboard41). The `ClubPage` on this PR is about the [Open Source Clubs landing page](https://m-branson.github.io/coss-os-ui/#artboard30).

**Comment by akatsoulas** [comment]
I consider the landing page page you are referring to, as the landing page for the opensource clubs and I would add it inside the open source clubs app. For example, if we had a completely different app leveraging the CoSS utilities like moderator, it would have a completely different landing page. i don't believe that we can have the "groups and gatherings" landing page for every site running inside CoSS.

**Comment by comzeradd** [comment]
PR updated. Slightly refactored to meet the scheme suggested.

File: coss/home/migrations/0002_create_homepage.py [hunk 1]
```diff
@@ -1,59 +0,0 @@
-# -*- coding: utf-8 -*-
-from __future__ import unicode_literals
-
-from django.db import migrations
-
-
-def create_homepage(apps, schema_editor):
```
Comment (akatsoulas) [inline coss/home/migrations/0002_create_homepage.py]: FYI, this is a data migration that creates a few entries in the db. Are you sure that by deleting this won't affect the db state on the server?

**Comment by comzeradd** [comment]
I reverted the changes on `home` app, so we can use it for the top-level landing page.

**Pull request merged**